posthog/api/test/test_survey.py

```diff
@@ -2310,6 +2427,7 @@ def test_list_surveys_with_actions(self):
 
     @snapshot_postgres_queries
     def test_list_surveys(self):
+        self.maxDiff = None
```

Comment:
Might have to remove this once tests are passing
```suggestion
        self.maxDiff = None
```